<launch>
    <node
            pkg="navigation"
            exec="KeybrdCtrl"
            name="KeybrdCtrl"
            launch-prefix="mate-terminal --tab --title='key' -x" />
    <node
            pkg="navigation"
            exec="RandomCtrl"
            name="RandomCtrl"
            launch-prefix="mate-terminal --tab --title='rnd' -x" />
    <node
            pkg="navigation"
            exec="ReactvCtrl"
            name="ReactvCtrl"
            launch-prefix="mate-terminal --tab --title='REA' -x" />
    <node
            pkg="navigation"
            exec="SquareCtrl"
            name="SquareCtrl"
            launch-prefix="mate-terminal --tab --title='sqr' -x" />
    <node
            pkg="navigation"
            exec="actuator"
            name="actuator"
            launch-prefix="mate-terminal --tab --title='ACT' -x"
            args="--topicpub /PioneerP3DX/cmd_vel" />
</launch>
